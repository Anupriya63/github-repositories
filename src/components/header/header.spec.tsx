import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store'; 
import Header from './header';

const initialState = {
    repos: [],
    error: '',
    notFound: false,
};

const mockStore = configureStore();
const defaultStore = mockStore(initialState);


describe('Header', () => {

    it('render header for results', () => {
        const header = shallow(
            <Provider store={defaultStore}>
                <Header />
            </Provider>);
        expect(header).toMatchSnapshot();
    });
});
