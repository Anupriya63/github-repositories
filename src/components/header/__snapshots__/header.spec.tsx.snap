// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render header for results 1`] = `ShallowWrapper {}`;
